test/extended/networking/util.go

```diff
@@ -120,6 +120,22 @@ func waitForEndpoint(c kclientset.Interface, ns, name string) error {
 	return fmt.Errorf("Failed to get endpoints for %s/%s", ns, name)
 }
 
+func waitForNetNamespace(nsName string) error {
```

Comment:
it would be simpler to just move the loop into makeNamespaceMulticastEnabled()

Reply:
So I forgot this was open, and did that in #20264 .  There's no harm in looping if it's a general helper.  

I don't care which option we go with, but I need one of them tagged soon to stop this flake :)